packages/eds-core-react/src/components/Button/Button.docs.mdx

```diff
@@ -132,4 +132,60 @@ The `Button.Group` can be displayed vertically using the `vertical` prop.
 `Button.Group` can also be used to create a split button. <br />
 The dropdown can change the button action (as in this example) or be used to immediately trigger a related action.
 
-<Story id="inputs-button--group-split" />
\ No newline at end of file
+<Story id="inputs-button--group-split" />
+
+## Button.Toggle
+Is used to group related **options**.
+
+### Usage
+
+The `Button.Toggle`
+<ul>
+  <li>emphasizes groups of related <code>Toggle</code> buttons,</li>
+  <li>controls the selected state of its child buttons when given its own <code>selectedIndexes</code> prop.</li>
+</ul>
+
+```tsx
+import { Button } from '@equinor/eds-core-react'
+import { save } from '@equinor/eds-icons'
+
+<Button.Toggle aria-label="actions">
+  <Button aria-label="save action">
+    <Icon data={save} title="Ghost icon save"></Icon>
+  </Button>
+  <Button aria-label="save action">
+    <Icon data={save} title="Ghost icon save"></Icon>
+  </Button>
+  <Button aria-label="save action">
+    <Icon data={save} title="Ghost icon save"></Icon>
+  </Button>
+</Button.Toggle>
+```
+
+### Accessibility
+
+#### ARIA
+<ul>
+  <li>Button.Toggle sets <code>aria-pressed="bool"</code> according to the button state.</li>
+  <li>You should provide an accessible label with <code>aria-label="label"</code>, <code>aria-labelledby="id"</code> or <code>&lt;label&gt;</code>.</li>
+</ul>
+
+## Examples
+
+### Exclusive selection
+
+Exclusive selection is a default sate, which means that by, selecting one option deselects any other.
+
+<Story id="inputs-button--toggle-exclusive-selection" />
+
+### Multiple selection
+
+Multiple selection allows for logically-grouped options, to have multiple options selected, by using the `multiple` prop.
+
+<Story id="inputs-button--toggle-multiple" />
+
+### Controlled
+
+To enforce at least one button to be active, you can adapt your `handleChange` function with `selectedIndexes` prop.
```

Comment:
I think the text from the deleted "enfored" section has ended up as the text for the "controlled" section, but that has not been implemented here

Reply:
is this not enough as a dummy story?